fbw-a380x/src/systems/extras-host/index.ts

```diff
@@ -61,6 +64,14 @@ class ExtrasHost extends BaseInstrument {
         this.versionCheck = new VersionCheck(this.bus);
         this.keyInterceptor = new KeyInterceptor(this.bus, this.notificationManager);
 
+        this.remoteClient = new RemoteClient({
+            websocketUrl: () => `${getSimBridgeUrl('ws')}/interfaces/v1/remote-app`,
+            airframeName: 'A380-842',
+            clientName: 'A380X',
```

Comment:
```suggestion
            airframeName: 'A380-842',
            clientName: 'A380X',
```
See point about using either an existing config, or (plan) to implement the new VFS based solution with a TODO: